$OpenBSD: patch-tests_offsets_offsets_h,v 1.1 2013/03/25 07:59:20 jasper Exp $

64-bit time_t is assumed, otherwise the output doesn't match what's expected.

--- tests/offsets/offsets.h.orig	Sun Mar 24 23:50:02 2013
+++ tests/offsets/offsets.h	Sun Mar 24 23:50:06 2013
@@ -29,7 +29,7 @@ struct _OffsetsBasic {
   char dummy8;
   gsize field_size;
   char dummy9;
-  time_t field_time;
+  long field_time;
 };
 
 typedef enum {
